core/src/style/pointStyle.h

```diff
@@ -38,6 +39,8 @@ class PointStyle : public Style {
 
     virtual ~PointStyle();
 
+    auto& getMesh() const { return m_mesh; }
```

Comment:
Will be good to have function name consistent between `PointStyle` and `TestStyle`. `TestStyle` has a method called `mesh` and `PointStyle` has a method called `getMesh`.